docs/commerce/payment/payum_integration.md

```diff
@@ -0,0 +1,79 @@
+---
+description: Add new payment methods through Payum such as Stripe or PayPal.
+edition: commerce
+---
+
+# Payum integration
+
+[Payum](https://payum.gitbook.io/payum/) is a payment processing solution that simplifies the integration of various payment services like Stripe and PayPal into your application.
+These services provide security of online transactions, and allow you to accept multiple payment methods while ensuring a seamless experience for the customers.
+By configuring service gateways, mapping workflow actions and translating payment service names, you streamline the online payment process, and can offer a diverse payment experience.
+
+## General Payum configuration
+
+In your Payum configuration file, for example, `payum.yaml`, set up a payment service gateway by specifying the factory, credentials and other necessary settings.
+Replace `<service_identifier>` with a unique identifier for the payment service.
+
+```yaml
+payum:
+    gateways:
+        <service_identifier>:
+            factory: <gateway_factory>
+            # Add specific configuration fields for the gateway
+            credential_1: <credential_1_value>
+            credential_2: <credential_2_value>
+```
+
+## Workflow mapping
+
+In [[= product_name =]], the default payment workflow has certain places, such as `pending`, `failed`, `paid`, or `cancelled`, and their corresponding transitions.
+However, for your application to use other transitions and places, for example, `authorized`, `notified`, `refunded` etc., and to present them in the user interface, you need to:
+- override the default payment workflow
+- create a custom workflow and enable it by using semantic configuration
+
+For more information, see [Custom payment workflows](configure_payment.md#custom-payment-workflows).
+
+For these places to be supported by the Payum integration, you have to map Payum statuses on the existing or additional places in the workflow, for example:
+
+```yaml
+ibexa_connector_payum:
+    status_mapping:
+        refunded: cancelled
+        captured: pending
+        authorized: authorized
+[...]
+```
+
+## Payment service name translations
+
+Within the `ibexa_payment_type` namespace in your translation files, add translations for each payment service that you configure.
+For language translations of payment service names, structure the translation files as follows:
+
+```yaml
+ibexa:
+    payment_method:
+        type:
+            <service_identifier>:
```

Comment:
```suggestion
            <gateway_identifier>:
```
as services have specific meaning in symfony ecosystem, even more service_identifier